utils/utils.py

```diff
@@ -123,6 +124,19 @@ def untar(destination, source):
         raise RuntimeError('Error occured while untaring "%s": %s' % (source, str(err)))
 
 
+def compress(source, destination):
```

Comment:
A very bad name.  First, it's far to generic, and second, no compression is occurring.  "append_to_tar()" or something like that is better.

Also, the legacy Unix convention for command/libraries with a "source" and "destination" concept is to put the destination FIRST.   You have mixed modes throughout, since this is a new call, please follow the convention.